core/src/typecheck/unif.rs

```diff
@@ -1599,7 +1599,7 @@ impl Unify for UnifRecordRows {
 }
 
 #[derive(Clone, Copy, Debug)]
-enum RemoveRowError {
+pub enum RemoveRowError {
```

Comment:
Is there a reason to make this symbol public? Public means that it's accessible to other crates. I think most of those symbols are specific to `nickel-lang-core`, or even to `typecheck`.

Reply:
In the mod.rs file, rust does not know what is RemoveRowError. It says it is not accessible.

Reply:
In which file? `typecheck/mod.rs`?